ramls/types/packages/packagePutData.json

```diff
@@ -15,7 +21,7 @@
     "attributes": {
       "description": "Data type attributes",
       "type": "object",
-      "$ref": "packagePutDataAttributes.json"
+      "$ref": "packageDataAttributes.json"
```

Comment:
Interesting other approach that you mentioned for this to `keep packagePutDataAttributes.json with "additionalProperties": true` -- we would explicitly only define the fields that are updated in that case - but this is probably fine and consistent with Provider